terra/projects/permissions.py

```diff
@@ -128,3 +129,9 @@ class IsModelVersionModelPublic(permissions.BasePermission):
 
     def has_object_permission(self, request, view, obj):
         return obj.model.is_public
+
+class HasBetaAccess(permissions.BasePermission):
+    message = 'This feature is only available to beta users.'
```

Comment:
This custom messages are not working right now as we are using an old DRF version, maybe we should upgrade later?